### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by DavidTTran**
General Refactor
- Added `Go Back` links to pages
- All passwords use a password field.
- Refactor NavBar for DRY
- Added NavBar button for Admins to view All Merchants
- `flash[:success]` in place of `flash[:notice]` 
- Merchant/orders show order status, item_orders can be fulfilled only if the item is pending and merchant has enough stock.

### CODE REVIEW

File: app/views/items/edit.html.erb [hunk 1]
```diff
@@ -19,4 +19,5 @@
 
     <%= submit_tag 'Update Item' %>
   <% end %>
+  <%= link_to "Go Back", request.referer %>
```
Comment (colintalex) [inline app/views/items/edit.html.erb:22]: Nice

File: app/views/merchants/edit.html.erb [hunk 2]
```diff
@@ -18,4 +18,5 @@
 
     <%= submit_tag 'Update Merchant' %>
   <% end %>
+  <%= link_to "Go Back", request.referer %>
```
Comment (colintalex) [inline app/views/merchants/edit.html.erb:21]: These are simply helpful, not sure why the user stories don't ask for these

**Review by colintalex — APPROVED** [review]
Looks good. thanks for doing those Go Back links and cleaning up the flashes

File: app/controllers/merchant/items_controller.rb [hunk 3]
```diff
@@ -29,10 +29,10 @@ def add_item
     @merchant = Merchant.find(current_user[:merchant_id])
     item = @merchant.items.new(item_params)
     if item.save
-      flash[:notice] = "#{item.name} has been added to your catalog."
+      flash[:success] = "#{item.name} has been added to your catalog."
```
Comment (Maxwell-Baird) [inline app/controllers/merchant/items_controller.rb:32]: Thank for change these to fit in style

**Pull request merged**